framework/test/jest/unit/specs/modules/chain/synchronizer/utils.spec.js

```diff
@@ -0,0 +1,75 @@
+/*
+ * Copyright © 2019 Lisk Foundation
+ *
+ * See the LICENSE file at the top-level directory of this distribution
+ * for licensing information.
+ *
+ * Unless otherwise agreed in a custom licensing agreement with the Lisk Foundation,
+ * no part of this software, including this file, may be copied, modified,
+ * propagated, or distributed except according to the terms contained in the
+ * LICENSE file.
+ *
+ * Removal or modification of this copyright notice is prohibited.
+ */
+
+'use strict';
+
+const {
+	restoreBlocks,
+} = require('../../../../../../../src/modules/chain/synchronizer/utils');
+
+describe('#synchronizer/utils', () => {
+	let blocksMock;
+	let processorMock;
+	const stubs = {};
+
+	beforeEach(async () => {
+		blocksMock = {
+			getTempBlocks: jest.fn(),
+		};
+
+		processorMock = {
+			processValidated: jest.fn(),
+		};
+
+		stubs.tx = jest.fn();
+	});
+
+	describe('restoreBlocks()', () => {
+		it('should return true on success', async () => {
+			// Arrange
+			const blocks = [{ id: 'block1' }, { id: 'block2' }];
+			blocksMock.getTempBlocks = jest.fn().mockReturnValue(blocks);
+
+			// Act
+			const result = await restoreBlocks(blocksMock, processorMock, stubs.tx);
+
+			// Assert
+			expect(blocksMock.getTempBlocks).toHaveBeenCalledWith(stubs.tx);
+			expect(processorMock.processValidated).toHaveBeenCalledTimes(2);
```

Comment:
probably these cases are worth to have another `it` phrase to describe?

Reply:
Agree, makes sense, thanks!